lua/lspsaga/provider.lua

```diff
@@ -236,6 +241,17 @@ function Finder:apply_float_map()
     { self.bufnr, "n", action.scroll_up, ":lua require'lspsaga.provider'.scroll_in_preview(-1)<CR>" },
   }
 
+  for i = 1, self.key_index do
+    local first_def_uri_lnum = self.definition_uri ~= 0 and 3 or 5
+    local last_def_uri_lnum = 3 + self.definition_uri - 1
+    local key = tostring(i)
+    local line = i + first_def_uri_lnum - 1
+    if line > last_def_uri_lnum then
+      line = line + 3
+    end
+    table.insert(keymaps, { self.bufnr, "n", key, ":call cursor(" .. tostring(line) .. ", 2)<CR>" })
```

Comment:
doesn't the new 0.7 api support binding functions? perhaps here can be `function() vim.fn.cursor(line, 2) end` for the sake of readability?